### DISCUSSION TITLE
Make the Actions workflow column wider

### DISCUSSION THREAD

**Discussion by lloydjatkinson** [community/community#5234] [Actions]
Currently the CSS column that displays workflow job names is probably too narrow for a lot of users. For example, I prefer slightly longer names.

This is what it looks like currently:
![image](https://user-images.githubusercontent.com/5633938/130337040-b6d325c9-937d-47b5-a12f-ebe62a7263e1.png)

I suggest it be made wider like this:
![image](https://user-images.githubusercontent.com/5633938/130336993-f329d376-e51e-49c1-9b12-be81aedd3d7c.png)

Now I know a lot of the time people have simple names like "Build" or just "CI" but in environments where there are several jobs to support slightly more complicated workflows having this column wider would be a huge benefit.

**Comment by dclayton-godaddy** [comment]
I would prefer to have the whole UI responsive to consume more screen if available.
